Oneirocom/Magick
# Nx fixes

## What Changed:

Upgraded NX to latest version.  Had to fix a bunch of typescript issues.  Things are much faster now. Rebuilds should be speedy, when you make an update to the client it hot reloads again.  If you make changes to a plugin, it rebuilds quickly.

## How to test:

Should all just work.  Not many functional changes. Almost all build changes and typescript error fixes. Try it out by saving a file in a few different places and checking build time etc.
